com.kristianduske.TrenchBroom.json

```diff
@@ -0,0 +1,144 @@
+{
+    "app-id": "com.kristianduske.TrenchBroom",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "trenchbroom.sh",
+    "rename-desktop-file": "trenchbroom.desktop",
+    "rename-icon": "trenchbroom",
+    "finish-args": [
+        "--device=dri",
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=ipc",
+        "--filesystem=host",
+        "--filesystem=home"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/pkgconfig",
+        "/share/aclocal",
+        "/man",
+        "/share/man",
+        "/share/gtk-doc",
+        "*.la",
+        "*.a"
+    ],
+    "build-options": {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g",
+        "env": {
+            "V": "1",
+            "DISPLAY": ":10"
+        }
+    },
+    "modules": [
+        "shared-modules/glu/glu-9.0.0.json",
+        "shared-modules/gtk2/gtk2.json",
+        {
+            "name": "wxwidgets",
+            "buildsystem": "autotools",
+            "config-opts": [
+                "--with-opengl",
+                "--with-cxx=17",
+                "--with-gtk=2",
+                "--disable-shared",
+                "--disable-precomp-headers",
+                "--with-libpng=builtin",
+                "--with-libtiff=builtin",
+                "--with-libjpeg=builtin"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/wxWidgets/wxWidgets/releases/download/v3.1.1/wxWidgets-3.1.1.tar.bz2",
```

Comment:
Why not *3.1.2*?

Reply:
Same reason as before, we are likely moving to QT so we didn't immediately jump onto 3.1.2.